packages/desktop-client/src/style/themes/light.ts

```diff
@@ -37,7 +37,7 @@ export const tableBorderSelected = colorPalette.purple500;
 export const tableBorderHover = colorPalette.purple400;
 export const tableBorderSeparator = colorPalette.navy400;
 export const tableRowBackgroundHighlight = colorPalette.blue150;
-export const tableRowBackgroundHighlightText = colorPalette.navy700;
+export const tableRowBackgroundHighlightText = pageText;
```

Comment:
Incorrect color used in the table (hover is totally broken right now; we have a separate issue to address that).